### ISSUE TITLE
Query on pretrained conditional model

### ISSUE THREAD

**Issue by SamLynnEvans** [yaxingwang/Transferring-GANs#4]
Hi Yaxing,

I was just wondering how you pretrained your AC-GAN;

Did you train it on ImageNet with all 1000 classes?

Does that then mean you always have to prefix a one-hot vector of length 1000 to the noise vector even when fine-tuning on a small number of classes?

If not, do you mind letting me know how this process works?

Thanks in advance!

**Comment by yaxingwang** [comment]
Hi, I use the pre-trained model from the unconditional model. Especially, the weight is transformed to the conditional model except for the weight for batchnorm,  which you need to copy 10 times(if you have 10 classes when you use ACGAN) of the weight of bachnorm of the unconditional model to every class. In our paper. I have pointed it, Please check it.
